eng/Packages.Data.props

```diff
@@ -102,7 +102,7 @@
     <PackageReference Update="Azure.ResourceManager" Version="1.3.1" />
 
     <!-- Other approved packages -->
-    <PackageReference Update="Microsoft.Azure.Amqp" Version="2.5.10" />
+    <PackageReference Update="Microsoft.Azure.Amqp" Version="2.5.12" />
```

Comment:
Beat me to it.    Should we consider updating the SB and EH change logs to include this?

Reply:
Hmm I usually update SB when releasing as opposed to doing it for each change.

Reply:
All good.   I'll throw the EH updates in the other open PR that I've got.